.circleci/config.yml

```diff
@@ -79,8 +82,18 @@ commands:
           path: ./dist
           prefix: ./dist
   build-checks:
+    parameters:
+      datadog-test-service:
+        type: string
+        default: ""
     steps:
       - build
+      - run: |
+          echo "export DD_SERVICE=<<parameters.datadog-test-service>>" >> $BASH_ENV
+          echo "export DD_ENV=ci" >> $BASH_ENV
```

Comment:
Do you think that we should just add this to the CircleCI project config? I like the other one being a string parameter though, good stuff.